nemo_text_processing/text_normalization/es/graph_utils.py

```diff
@@ -40,6 +40,10 @@
 fem_hundreds = hundreds @ pynini.cdrewrite(pynini.cross("ientos", "ientas"), "", "", NEMO_SIGMA)
 
 
+ES_MINUS = pynini.union("menos", "Menos", "MENOS").optimize()
```

Comment:
Any informal form of these that we should consider?

Reply:
should be fine as is -- especially since this PR is focused more on adding coverage of capitalization specifically and not necessarily general improvements :)